include/solvers/mpm_base.h

```diff
@@ -27,6 +27,9 @@
 #include "particle.h"
 #include "vector.h"
 
+// CSV-parser
+#include "csv.h"
```

Comment:
Should be included in `io` class not mpm base

Reply:
Got it!

Reply:
Should we move `json` and `catch` to their own respective folders?